ArjunaJTS/integration/src/test/java/com/arjuna/ats/jta/distributed/SimpleIsolatedServers.java

```diff
@@ -128,7 +129,11 @@ private void boot(int index) throws SecurityException, NoSuchMethodException, In
                 .getName(), contextClassLoader);
         IsolatableServersClassLoader classLoader = new IsolatableServersClassLoader(SimpleIsolatedServers.class.getPackage().getName(), null,
                 classLoaderForTransactionManager);
-        localServers[index] = (LocalServer) classLoader.loadClass("com.arjuna.ats.jta.distributed.server.impl.ServerImpl").newInstance();
+        try {
+	    localServers[index] = (LocalServer) classLoader.loadClass("com.arjuna.ats.jta.distributed.server.impl.ServerImpl").getDeclaredConstructor().newInstance();
+        } catch ( InvocationTargetException e ) {
+	    throw new RuntimeException( e );
+        }
```

Comment:
Since there are already quite a few exceptions listed in the method signature, why can’t we add one more?